server/src/main/java/org/apache/druid/server/coordinator/helper/DruidCoordinatorRuleRunner.java

```diff
@@ -141,13 +139,8 @@ public DruidCoordinatorRuntimeParams run(DruidCoordinatorRuntimeParams params)
 
   private Set<DataSegment> determineOvershadowedSegments(DruidCoordinatorRuntimeParams params)
```

Comment:
Looks like this method is the same as `getFullyOvershadowedSegments()`, can they be merged?

Reply:
ok, made `determineOvershadowedSegments` static in `ImmutableDruidDataSource` and removed the one from here